buildpacks/python/python/build.go

```diff
@@ -112,40 +112,49 @@ func (b Build) getFuncYamlOptions(appPath string) []libcnb.Label {
 }
 
 func (b Build) optionsToLabels(options knfn.Options) []libcnb.Label {
-	var requestsJson []byte
-	var limitsJson []byte
 	labels := []libcnb.Label{}
 
-	scaleJson, err := json.Marshal(options.Scale)
-	if err != nil {
-		b.Logger.Bodyf("unable to marshal func.yaml options.Scale")
+	if options.Scale != nil {
```

Comment:
instead of returning `nil`, can 0 be a default we can take from the "I am reading the `scale` option"?

Reply:
Good point. Though instead of returning 0, I actually want to either not provide that label at all or fail the build. (Applies to all the other fields)

Reply:
Sure, at the end of the day that's a optional field in the Serving object